Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,44 @@
-﻿namespace Logger;
+﻿using System;
+using System.Linq;
+
+namespace Logger;
 
 public static class BaseLoggerMixins
 {
-    
+    public static void Error(this BaseLogger log, string message, params object[] args)
```

Comment:
```suggestion
    public static void Error(this BaseLogger logger, string message, params object[] args)
```
A better name here would be 'logger' since that's what you're passing in, a type of logger